### PR TITLE
Add time zone selection options

### PR DESCRIPTION
**Pull request by freddy2659**

### CODE REVIEW

**Comment by freddy2659** [comment]
Couldn't check due to auth redirection, but it should be working or close to what it's meant to be

File: client/src/component/ProfileWidget.js [hunk 1]
```diff
@@ -61,6 +61,16 @@ const ProfileWidget = (props) => {
...
+  const timeOptions = [];
+  for (var i = -12; i <= 14; i++){
+    if(i < 0) {
+      timeOptions.push({'value':i, 'text':"UTC" + i + ":00"});
+    } else if (i === 0) {
+      timeOptions.push({'value':i, 'text':"UTC Time"});
+    } else {
+      timeOptions.push({'value':i, 'text':"UTC+" + i + ":00"});
+    }
+  }
```
Comment (jassalss) [inline client/src/component/ProfileWidget.js:73]: My resources are saying there is only 24/25 times zones not 26/27

File: client/src/component/AvailabilityWidget.js [hunk 2]
```diff
@@ -155,12 +154,12 @@ const AvailabilityWidget = (props) => {
...
-                        <Grid item sm>
+                        <Grid item sm={12}>
                             <Typography alight="left" variant="subtitle2">
                                 Available days:
                             </Typography>
                         </Grid>
-                        <Grid item sm>
+                        <Grid item sm={12}>
                             <ButtonGroup size="large" color="primary">
                                 {Object.keys(daysSelected).map((day) => (
```
Comment (jassalss) [inline client/src/component/AvailabilityWidget.js:165]: FormControlLabels inside ButtonGroup don't contains key={} props

**Review by jassalss — CHANGES_REQUESTED** [review]
Just two or three minor errors otherwise looks nice

**Pull request closed without merging**